cumulusci.yml

```diff
@@ -836,6 +845,13 @@ flows:
                     api_names: QA,Gift_Entry
 
 
+    deploy_unmanaged:
+        description: Override to the deploy_unmanaged flow to force a mock apex class to be deployed that will prevent compilation errors when the packaged code references file-based apex in the core stack for instrumentation purposes.
+        steps:
+            2.1:
+                when: not org_config.namespace
+                task: deploy_core_mock_classes
```

Comment:
The condition is correct and skip the mock class deployment. However, in the general flow `step 1) depenedices` `task 2: deploy_pre`, all files in the `unpakcaged/pre` are being deployed. This result `SfdoLogUtils.cls` being deployed regardless of the namespace.